build.sbt

```diff
@@ -313,6 +313,8 @@ lazy val node = (project in file("node"))
         Cmd("FROM", dockerBaseImage.value),
         ExecCmd("RUN", "apt", "update"),
         ExecCmd("RUN", "apt", "install", "-yq", "openssl"),
+        ExecCmd("RUN", "apt", "install", "-yq", "openssh-server"),
+        ExecCmd("RUN", "apt", "install", "-yq", "procps"),
```

Comment:
You should turn those two new into one `RUN`. This creates two unnecessary layers.

Reply:
why not `RUN apt update && apt install -yq openssl openssh-server procps`

Reply:
It's easier to tell which command failed, for one.